pretrain/model.py

```diff
@@ -16,18 +16,21 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 # DEALINGS IN THE SOFTWARE.
 
-from transformers import GPT2Config, GPT2LMHeadModel, AutoTokenizer
+from transformers import GPTNeoXConfig, GPTNeoXForCausalLM, AutoTokenizer
 
-config = GPT2Config()
+config = GPTNeoXConfig()
 
 
 def get_model():
```

Comment:
The miner is a template, but it looks like this will "break" the ability to train at 772. Should we have another "get_772_model()" method or something? Also fine to keep as-is though we may want to add a comment

Reply:
GPTNEeoX is supported for both cases right? Is there a reason it would break the ability to train?

Or did I just choose too big values for the default here you mean? I can double check the size.

Reply:
Updated to target close to 772 parameters and added a comment.